### DISCUSSION TITLE
Custom "New entry" defaults for custom sources

### DISCUSSION THREAD

**Discussion by adrienpicard** [craftcms/cms#13390] [Ideas] [answered]
I'm using custom sources to break down indexes by Entry Type within the same section, and it'd be great to define default settings (section and/or entry type) for the "New Entry" button as well.

I'm aware it's specific case, but at the moment clicking "New Entry" from a custom source lists all sections available and users might find that confusing.

Possibly related to https://github.com/craftcms/cms/discussions/11499

**Comment by brandonkelly** [comment, marked as answer]
Implemented for Craft 4.5! (#13404)

**Reply by adrienpicard** [reply]
Amazing! Thank you 🙏

**Reply by brandonkelly** [reply]
Craft 4.5.0 is out with this!
